Week1/homework/js-exercises/Error debugging.js

```diff
@@ -0,0 +1,7 @@
+// console.log('I'm awesome');
+//  changes single queto to double queto
+console.log("I'm awesome");
+
+// or 
+console.log('I\'m awesome');
+console.log('I`m awesome');
```

Comment:
This last one is not really correct, because the output would not be the same. the ` is a different character then the '. You already solved the exercise with the first two answers!